x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/autoscaling/MlMemoryAutoscalingDecider.java

```diff
@@ -945,4 +935,13 @@ private Long getAnomalyMemoryRequirement(String anomalyId) {
     private Long getAnomalyMemoryRequirement(PersistentTasksCustomMetadata.PersistentTask<?> task) {
         return getAnomalyMemoryRequirement(MlTasks.jobId(task.getId()));
     }
+
+    private static List<Long> computeJobSizes(List<String> unassignedJobs, Function<String, Long> sizeFunction) {
```

Comment:
@thecoop 

> What automations are done on this PR?

This one I extracted by hand and moved it to a normal loop since it came out wrong from the automatic refactoring in both spots (since we were using `Collectors.toList()` with assumption that it produces a mutable list so moving to `toList()` broke things.

Rest I think was all automatic.